flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/source/abilities/SupportsDynamicFiltering.java

```diff
@@ -60,9 +60,16 @@
 public interface SupportsDynamicFiltering {
 
     /**
-     * Applies the candidate filter fields into the table source, and return the accepted fields.
-     * The data corresponding the filter fields will be provided in runtime, which can be used to
-     * filter the partitions or the input data.
+     * Return the filter fields this partition table source supported. This method is used in
+     * planner, which can be used to judge whether the source table is a suitable partition table
+     * for filtering partitions.
```

Comment:
/**
     * Returns the filter fields this partition table source supported. This method can tell the
     * planner which fields can be used as dynamic filtering fields, the planner will pick some
     * fields from the returned fields based on the query, and create dynamic filtering operator.
     */

Reply:
> /** * Returns the filter fields this partition table source supported. This method can tell the * planner which fields can be used as dynamic filtering fields, the planner will pick some * fields from the returned fields based on the query, and create dynamic filtering operator. */

Done!